btk/draw_blends.py

```diff
@@ -356,6 +355,7 @@ def render_mini_batch(self, blend_list, psf, wcs, survey, noise_seed, extra_data
             )
             blend_image_multi = np.zeros((len(survey.filters), pix_stamp_size, pix_stamp_size))
             for b, filt in enumerate(survey.filters):
+                noise_seed = self.rng.integers(MAX_SEED_INT)
```

Comment:
Thanks @thuiop for starting this PR. Unfortunately, I think that the way you wrote it will lead to issues with multiprocessing since the different processes will access `self.rng` in non-deterministic order. So that the `noise_seed` used in each process will depend on which one is "faster".